modules/reference/pages/admin/cobbler.adoc

```diff
@@ -1,8 +1,7 @@
 [[s3-sattools-config-cobbler]]
 = Cobbler
 
-On the menu:Admin[Manager Configuration > Cobbler] page you can run the Cobbler synchronization by clicking btn:[Update].
+The menu:Admin[Manager Configuration > Cobbler] page allows you to run a Cobbler synchronization.
+You can repair or rebuild the contents of [path]``/srv/tftpboot`` or [path]``/srv/www/cobbler`` after a manual modification of the Cobbler setup.
```

Comment:
sth completely different... this reminds me.  We should either add a trailing slash to a directory path (/srv/tftpboot/) or put the literal word "directory" there
```
the [path]``/srv/tftpboot`` directory
```
(to be adjusted to the actual phrase, of course)

Reply:
or both?